.travis.yml

```
@@ -19,9 +19,12 @@ matrix:
     - python: nightly
 
 install:
-  - pip install --process-dependency-links .[dev]
-  - pip install codacy-coverage
-  - pip install pytest-cov
+  - pip install pip -U
+  - wget https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py
+  - python get-poetry.py --preview -y
+  - source $HOME/.poetry/env
+  - poetry install -v
+  - poetry add codacy-coverage pytest-cov
```

Should we instead maintain a CI-specific set of dependencies? We can install them with Poetry's [support for "extras"](https://poetry.eustace.io/docs/pyproject/#extras). Example:

```toml
[tool.poetry.extras]
ci = ["code-coverage", "pytest-cov"]
```

```suggestion
  - poetry install -v -E ci
  - poetry add codacy-coverage pytest-cov
```

---

That's a great idea and I'll try to look into that tonight.